frontend/src/components/ingress/Details.tsx

```diff
@@ -74,7 +75,8 @@ export default function IngressDetails() {
               columns={[
                 {
                   label: t('Host'),
-                  getter: (data: IngressRule) => data.host || '*',
+                  getter: (data: IngressRule) =>
+                    <Link routeName={`${data.host}`}>{data.host}</Link> || '*',
```

Comment:
The routeName is supposed to be a route's name from the object in router.tsx.
Also you need to validate whether the host is an actual URL before adding it as a link prop.